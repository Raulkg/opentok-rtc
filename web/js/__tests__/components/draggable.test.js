var draggable = require('../../components/draggable.js');
test('snapshot for draggable component', () => {
  expect(draggable).toMatchSnapshot();
});