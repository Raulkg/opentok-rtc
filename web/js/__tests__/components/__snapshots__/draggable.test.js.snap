// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for draggable component 1`] = `
Object {
  "Draggable": Object {
    "CLICK_THRESHOLD": 10,
    "DRAG_TIMEOUT": 200,
    "off": [Function],
    "on": [Function],
  },
}
`;
